### PR TITLE
added docs for wasm and ammo

### PR DESCRIPTION
**Pull request by c0debon**

### CODE REVIEW

File: content/en/user-manual/assets/wasm-modules.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: Wasm Modules
+template: usermanual-page.tmpl.html
+position: 17
+---
+
+Wasm Modules (also known was WebAssembly Modules) contain compiled executable code for the web.
+
+A wasm module comprises three parts:
+* the binary executable file
+* the javascript glue code file
```
Comment (willeastcott) [inline content/en/user-manual/assets/wasm-modules.md]: javascript -> JavaScript

File: content/en/user-manual/assets/wasm-modules.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* an optional fallback asm.js
+
+These files can be added to the project either by dragging and dropping the files into the Assets Panel or by selecting 'Upload' from the Assets Panel context menu.
+
+## Wasm Module Properties
+
+Once the files have been added to the project, select the Wasm Module to display it's properties in the Inspector Panel:
```
Comment (willeastcott) [inline content/en/user-manual/assets/wasm-modules.md]: it's -> its

File: content/en/user-manual/assets/wasm-modules.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+![Wasm Module Properties][1]
+
+### Name
+Name must match the module name defined in the glue and fallback script. This name is used to instantiate the module at load time.
+
+### Glue script
+This is the javascript glue code required to execute wasm code.
+
+### Fallback script
+This is the fallback asm.js script to use when WebAssembly is not supported.
```
Comment (willeastcott) [inline content/en/user-manual/assets/wasm-modules.md]: Is this required? What if I don't specify it, not caring about old browsers that don't support Wasm?

File: content/en/user-manual/physics/physics-basics.md [hunk 4]
```diff
@@ -10,13 +10,15 @@ PlayCanvas provides the [rigidbody][2] and [collision][3] components to set up p
...
 
-By default, a new PlayCanvas project will have physics disabled in the Scene Settings panel:
+By default, a new PlayCanvas project does not include ammo.js modules. This is because ammo.js weighs in at several hundred kilobytes and your app should not have to load this library if it is not needed.
+
+You can import ammo.js modules into your project using the import button on the Scene Settings panel:
 
 ![Physics Settings][4]
 
-This is because ammo.js weighs in at several hundred kilobytes and your app should not have to load this library if it is not needed.
+This will import the default build of ammo.js provided by Playcanvas. However please note it is possible to compile your own version of ammo.js and add it to your project instead. For more information see [this page][11].
```
Comment (willeastcott) [inline content/en/user-manual/physics/physics-basics.md]: Playcanvas -> PlayCanvas

File: content/en/user-manual/physics/physics-migration.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+title: Updating ammo.js
+template: usermanual-page.tmpl.html
+position: 6
+---
+
+### Introduction
+
+Playcanvas has added support for the latest version of ammo.js.
```
Comment (willeastcott) [inline content/en/user-manual/physics/physics-migration.md]: Playcanvas -> PlayCanvas

File: content/en/user-manual/assets/wasm-modules.md (hunk 3 above)
Comment (c0debon) [reply content/en/user-manual/assets/wasm-modules.md]: It's optional, which I added to the description.

**Pull request merged**